spot-contracts/contracts/BondIssuer.sol

```diff
@@ -113,7 +124,35 @@ contract BondIssuer is IBondIssuer, OwnableUpgradeable {
 
     /// @inheritdoc IBondIssuer
     function isInstance(IBondController bond) external view override returns (bool) {
-        return _issuedBonds.contains(address(bond));
+        return (_activeBonds.contains(address(bond)) || _matureBonds.contains(address(bond)));
+    }
+
+    /// @inheritdoc IBondIssuer
+    /// @dev Reverts if none of the active bonds are mature.
+    function matureAll() external {
+        bool bondsMature = false;
+        uint256 activeCount_ = _activeBonds.length();
+
+        // NOTE: We traverse the active list in the reverse order as deletions involve
+        //       swapping the deleted element to the end of the list and removing the last element.
+        for (uint256 i = activeCount_; i > 0; i--) {
+            IBondController bond = IBondController(_activeBonds.at(i - 1));
+            if (bond.timeToMaturity() == 0) {
```

Comment:
```suggestion
            if (bond.timeToMaturity() <= 0) {
```
Common defensive programming technique (unless there's a gas reason not to use it).

Reply:
👍